iree/turbine/kernel/wave/utils/graph_utils.py

```diff
@@ -59,7 +61,10 @@ def is_removable_operator(node: fx.Node) -> bool:
 
         if (
             custom.users
-            or isinstance(custom, (Output, SetSymbol, SharedMemoryBarrier))
+            or isinstance(
+                custom,
+                (Output, SetSymbol, SetPrio, SharedMemoryBarrier, WorkgroupBarrier),
```

Comment:
We really need some trait for it :)

Reply:
will add in this PR and end it once and for all :D

Reply:
done